### ISSUE TITLE
SSL on browserhacks.com

### ISSUE THREAD

**Issue by MaverickMartyn** [4ae9b8/browserhacks#193]
I just wanted to point out that you should probably add SSL support on the website, considering that Google is moving towards displaying security warnings on sites without it.

You could consider using something like Let's Encrypt to get free SSL certificates.

**Comment by browserstrangeness** [comment]
Not a bad idea at all. Hugo, is this something you need to do at the source or something you are up for setting-up?

**Comment by laukstein** [comment]
browserhacks.com is build on GitHub Page. All that is needs to enable SSL is to go to https://github.com/4ae9b8/browserhacks/settings and check "Enforce HTTPS".

Documentation https://help.github.com/articles/securing-your-github-pages-site-with-https/

The problem is https://github.com/4ae9b8/browserhacks/issues/194 that no one is maintaining this project for years.

**Comment by browserstrangeness** [comment]
https://github.com/4ae9b8/browserhacks/settings error 404 (page not found) -- will check settings however.

**Comment by laukstein** [comment]
> https://github.com/4ae9b8/browserhacks/settings error 404 (page not found) -- will check settings however.

needed admin rights to access repo settings.

**Comment by browserstrangeness** [comment]
True enough - I may have access to content, but site 'hosting' changes -- doubtful.

**Comment by browserstrangeness** [comment]
You are correct, that is a different type of administrator that does that work, the site owner to be exact. Looking back, I did that on my mirror of my own testing page at https://browserstrangeness.github.io but I do not appear to have the rights to do that on this site.

**Comment by laukstein** [comment]
@browserstrangeness where is @4ae9b8 admin? He expected having admin rights for it.

**Comment by browserstrangeness** [comment]
Good question.

**Comment by XhmikosR** [comment]
@HugoGiraudel: can you enable https?

**Comment by KittyGiraudel** [comment]
Hey there. I’m no longer maintaining Browserhacks and am not part of the @4ae9b8 organisation anymore. That might be for @TimPietrusky I guess now. ;)

**Comment by laukstein** [comment]
@SaraSoueidan, do you have admin right fro this repo, can you add me too so I can review/accept pull request, add SSL, etc. progress this repo?

**Comment by MaverickMartyn** [comment]
It's good to see people still trying to keep this alive. :)

**Comment by browserstrangeness** [comment]
Actually so am I, it has been a while since there has been any real communication. I don't have the authority to add other users, probably not even the admin level. 

Updating information of hacks is different --> One of the issues I had in the past was that people would introduce pull requests to make changes that were not validated among all browsers, so not really smart to allow the pull request. There is much people need to know before making updates. 

One of the biggest mistakes is to post 'works on firefox 55' and then someone on ios says that it does not, which if you have been reading my notes, firefox and chrome on ios devices use the safari back end (probably due to a licensing issue with apple) -- Basically the hack does work properly... on the proper os. Some of them worked on linux and chromium due to the lack of additions they have that chrome includes being a proprietary browser. So it is not an easy job - but looks easy at first glance.

That all said, I am THRILLED people are interested in this site still - though not a large number I can assume that more are out there.

**Comment by laukstein** [comment]
@browserstrangeness, you simply use BrowserStack to test whatever on different browsers, versions, OSs, devices. Likely best is to clone this project or open another one that could be kept always up-to-date. This ... project is outdated.

**Comment by browserstrangeness** [comment]
That is true, but people haven't been doing that. That is what I am saying.

**Comment by browserstrangeness** [comment]
But yes, that is how I handle it -- so if people want to make updates, then the whole job must be completed. Not just one test with the browser they are using, but the complete testing of all major browsers listed on our site. The ios issue I mentioned above is one that would ruin the site since people would not be aware of the problem unless they had done this properly.

**Comment by KittyGiraudel** [comment]
Good morning to you all.

Let’s start with the obvious: this project is no longer actively maintained. I am also not convinced it really has to be anymore. Developers who still have to maintain backward compatibility with legacy systems can use the hacks listed on that website. Developers working on newer systems should have less and less needs for browser-specific hacks when building user interfaces.

I believe the decay of that project is not only the symptom of the lack of time from its maintainers. It’s also that such project has way less of a room in the current web development landscape than it did at the time of its creation.

If one wants to fork or create a similar one, basing their work on the Browserhacks database, be my guest. I just don’t think that’s particularly worth it. :)

**Comment by laukstein** [comment]
@HugoGiraudel, I agree there are way less incompatibilities and inconsistencies between browsers than used to be, but still it is and will stay relevant in most featured browsers. It likely would be best if could handle all with CSS `@supports`, `@supports not` and JS `CSS.supports()` instead, without targeting any specific browser.
